manifests/share.pp

```diff
@@ -3,9 +3,9 @@
 # This class setups up a config file to be included into smb.conf
 #
 define winbind::share (
-  $ensure_setting  = present,
-  $settings_hash   = undef,
```

Comment:
```puppet
$ensure_setting  = present,
$settings_hash   = undef,
```
These didn't pull from `params.pp` before so why do they now need to pull from hiera? Just trying to understand.